### PR TITLE
Fix/travis share build

### PR DESCRIPTION
**Pull request by anechifor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+stages:
+  - name: build
+  - name: test
+    if: commit_message !~ /\[skip tests\]/
+
+# This should not be required on community build
+before_install:
+  - "cp .travis.settings.xml $HOME/.m2/settings.xml"
+  - "chmod +x ./packaging/docker/tests/scripts/start-compose.sh"
```
Comment (cristina-sirbu) [inline .travis.yml:37]: Please push the scripts with the appropriate rights :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    - $HOME/.m2
+# the cache can grow constantly
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/alfresco/alfresco-share*
+  
+branches:
+  only:
+    - master
+    - /support\/.*/
+    - fix/Travis-share-build
```
Comment (cristina-sirbu) [inline .travis.yml:27]: Please use regex. :)

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+stages:
+  - name: build
+  - name: test
+    if: commit_message !~ /\[skip tests\]/
```
Comment (pieCit87) [inline .travis.yml:32]: In order to improve the mnemonic on commands, please use a naming convention similar to maven. eg: from [skip tests] to [skip-tests]

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+  only:
+    - master
+    - /support\/.*/
+    - fix/Travis-share-build
+    
+stages:
+  - build
+  - tests
+    if: commit_message !~ /\[skip-tests\]/
+  - release
```
Comment (alexbalmus) [inline .travis.yml:32]: In Bamboo we have the possibility to choose between "Engineering Release" (that pushes the distribution zip to Nexus) and "Company Release" (that pushes the Docker image to DockerHub).

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+      install: skip
+      script: travis_wait 30 bash _ci/source_clear.sh
+      
+    - name: "Push Image To Quay"
+      stage: build
+  
+    - name: "SHARE PO ADMIN-TOOLS"
+      stage: tests
+      jdk: openjdk11
+      install:
```
Comment (alexbalmus) [inline .travis.yml:69]: There are quite a few repetitive sections following. Please check if it's possible to rewrite using YAML anchors and aliases: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

File: _ci/.wss-unified-agent.config [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+ 
+##################################
+# Organization tokens:
+##################################
+apiKey=
+ 
+#userKey is required if WhiteSource administrator has enabled "Enforce user level access" option
+#userKey=
+   
+projectName=Aikau
```
Comment (alexbalmus) [inline _ci/.wss-unified-agent.config:134]: Is this only specific to Aikau?

**Pull request closed without merging**